CODEOWNERS

```diff
@@ -3,6 +3,9 @@
 /packages/dd-trace/src/appsec/ @DataDog/asm-js
 /packages/dd-trace/test/appsec/ @DataDog/asm-js
 
+/packages/dd-trace/src/ai_guard/ @DataDog/asm-js
+/packages/dd-trace/test/ai_guard/ @DataDog/asm-js
```

Comment:
This is adding as codeowner of this PR a team where the owner of the PR is not in. We can start like this now, but we should find a better way to do this, maybe creating a new GH team for this?

Reply:
Yes, this is a bit the chicken and the egg issue, even though the team was not assigned at least it got reviewed by the team :smile:

Reply:
Do you guys have a github team already ? I'm not happy being codeowner of this :/